mla/analysis.py

```diff
@@ -174,12 +362,15 @@ def get_ts(args):
                                              **kwargs)
 
             # Store the results in the output array
+                
             output['ts'] = -1*result.fun
+            if np.isnan(output['ts']):
+                output['ts'] = 0
```

Comment:
is there a better way to handle this?

Reply:
This part shouldn't exist. I ran into this and I have no idea why it returns nan sometimes. I add this for now to make some of my test works but it should not happen. I will look into it soon.